src/utils.js

```diff
@@ -0,0 +1,59 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+const Constant = require("./Constant");
+
+/**
+ * @param message:출력하고자 하는 문자열
+ * 문자열을 출력해주는 함수
+ */
+function print(message){
+    MissionUtils.Console.print(message);
+}
+
+/**
+ * @param message: 입력값앞에 출력할 message
+ */
+function scan(message){
+    let result = null;
+    MissionUtils.Console.readLine(message, (input) => {
+        result = input;
+    });
+    return result;
+}
+
+// 오름차순 정렬 함수
+function ascSort(a,b) {
+    return a-b;
+}
+
+function validateLength(numbers) {
+    if (numbers.length !== Constant.LOTTO_LENGTH) {
+        throw new Error("[ERROR] 로또 번호는 "+Constant.LOTTO_LENGTH+"개여야 합니다.");
+    }
+}
+
+function validateOverlap(numbers){
+    const set = new Set(numbers);
+    if(set.size !== Constant.LOTTO_LENGTH) throw new Error("[ERROR] 로또 번호가 중복되었습니다.");
+}
+
+function validateNumber(numbers) {
+    for(let i = 0 ; i < Constant.LOTTO_LENGTH ; i++){
+        if(numbers[i] <= 0 || numbers[i] >= 46) throw new Error("[ERROR] 로또 번호는 1~45사이어야 합니다.");
+    }
```

Comment:
```suggestion
    for(const number of numbers){
        if (number <= 0 || number >= 46) throw new Error("[ERROR] 로또 번호는 1~45사이어야 합니다.");
    }
```
`for..of` 를 쓰면 간결하게 쓸 수 있어요!

Reply:
for..of 라는 method를 공부해보긴했는데 사용하는 습관이 베이지 않았었는데, 확실히 코드를 이렇게 비교해서 보니까 더 간결하고 보기 좋네요 ..! 이번 4주차 과제에서 적용해보면 좋을 것 같다는 생각이 드네요 ! 리뷰 감사합니다 !! ㅎ 👍